cli/src/main/kotlin/commands/Constants.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.commands
+
+const val CONFIGURATION_OPTIONS = "Configuration Options"
```

Comment:
I'd prefer to have a common prefix for these global constants, like e.g. "CLI_" (even though it's in the `cli`module), or maybe by just swapping the order and making `OPTIONS_` the prefix instead of suffix.

Reply:
Done, I have used `OPTION_GROUP_...`.